Dockerfile

```diff
@@ -1,34 +1,57 @@
-#
+### ### ### ### ### ### ### ### ###
 # Builder layer
-#
+
 FROM node:6-alpine as ghost-builder
-RUN npm install --loglevel=error -g ghost-cli
 
-ENV GHOST_VERSION 1.17.1
-ENV GHOST_INSTALL /var/lib/ghost
-ENV GHOST_CONTENT /var/lib/ghost/content
-ENV GHOST_USER node
+RUN \
+    apk update && apk upgrade                       	&& \
+    apk add --no-cache wget unzip ca-certificates   	&& \
```

Comment:
Has written in the comment. I didn't get the target of these packages.

Reply:
Oh yes, these are needed when I override casper theme with my own theme. 
Useless in this case.

Reply:
Ok. And, even with your theme, do you need these package inside the Docker?

Reply:
Yes, I could use COPY and git clone the repo on the host but I prefer doing this via docker. 

With the fact it's not in the final image, I see no harm.